Revision en docs (#1323)
mercadopago/devsite-docs: guides/online-payments/marketplace/checkout-api/introduction.en.md

Mercado Pago allows you to collect payments on behalf of the sellers of your platform and optionally charge a commission for the transaction.

When a payment is generated, the amount is immediately divided between the seller’s account and your account, in case you charge a fee.

> _Clarification: The payment split can only be done between two Mercado Pago accounts (Marketplace and the seller associated to it), no more._

Reviewer: ```suggestion
**Clarification**: The payment split can only be done between two Mercado Pago accounts (Marketplace and the seller associated to it).
```

> _Clarification: The payment split can only be done between two Mercado Pago accounts (Marketplace and the seller associated to it), no more._

Reviewer: @romisoto - Romi, será que aqui não ficaria melhor uma Note?

> Your [credentials]([FAKER][CREDENTIALS][URL]) are the **keys we provide so you can configure your integrations.**. 

Reviewer: ```suggestion
Your [credentials]([FAKER][CREDENTIALS][URL]) are the **keys we provide for you to configure your integrations.**. 
```
